modules/nf-core/pangolin/run/tests/main.nf.test

```diff
@@ -30,6 +33,42 @@ nextflow_process {
 
     }
 
+    test("sarscov2 genome [fasta] - update data") {
+        setup{
+            run("PANGOLIN_UPDATEDATA") {
+                script "../../updatedata/main.nf"
+                process {
+                    """
+                    input[0] = 'testdb'
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = [
+                            [ id:'test' ], // meta map
+                            [ file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true) ]
+                ]
+                input[1] = PANGOLIN_UPDATEDATA.out.db
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.report[0][1]).name,
```

Comment:
Is the created report unstable? can we maybe assert a bit more than just the name?

Reply:
Issue is that its not stable as its using a database that gets updated regulary. 
What would you suggest?

Reply:
maybe check whether the column headers stay the same? but if thats not possible we can also leave it like that

Reply:
ooh yeah totally, didn't think of that! Think with the nft-csv it's easy no.